react/PushClientButton/index.jsx

```diff
@@ -17,7 +16,7 @@ const ButtonClient = props => {
       onClick={onClick}
     >
       <figure>
-        <img width="32" height="32" src={DeviceIcon} />
+        <Icon icon="device-laptop" size="32" />
```

Comment:
Soooo it should be `devices-laptop`, right?

Reply:
Nop `device-laptop` is fine ;) https://docs.cozy.io/cozy-ui/react/#!/Icon/1

Reply:
I think it still works because the sprite was not re-generated, but it shouldn't, no?

Reply:
Oh come on! I didn't see my mistake in the filename :o. Thanks :)